packaging/linux/run_keybase

```diff
@@ -45,7 +45,7 @@ echo Launching keybase service...
 keybase service &>> "$logdir/keybase.service.log" &
 echo Mounting /keybase...
 kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
-  -bserver bserver.kbfs.keybase.io:443 /keybase &>> "$logdir/keybase.kbfs.log" &
+  -bserver bserver.kbfs.keybase.io:443 -log-to-file /keybase &>> "/dev/null" &
```

Comment:
Case (2) kind of makes me want to let output go to the terminal. That would also make it more likely that we catch bugs where someone logs something important to stderr, instead of using the proper logging calls.